generate_db.py

```diff
@@ -3,11 +3,11 @@
 import sys
 import os
 
-language = sys.argv[1]
+language = sys.argv[1] # the expected input is enwiki
 month = '2021-01'
 dump_dir = f"/public/dumps/public/other/clickstream/{month}/"
 clickstream_fn = f'clickstream-{language}-{month}.tsv.gz'
-database = os.path.expanduser(f'~/www/python/src/db/clickstream_{language[:2]}.db')
+database = os.path.expanduser(f'~/www/python/src/db/clickstream_{language[:2]}.db') # language.replace("wiki","")
```

Comment:
maybe the current approach is even better, but an alternative is to replace the wiki

Reply:
Replacing the wiki looks more readable. I'll fix it. Thanks!